demo/src/App.jsx

```diff
@@ -63,6 +64,16 @@ const App = () => (
           <option value="2" disabled>Option 2</option>
           <option value="3">Option 3</option>
         </Form.Select>
+        <Card.Item>
```

Comment:
Should just simply be `<Card />` it doesn't require any descendants like many of the other components.